src/main/cms/cms_menu_osd.c

```diff
@@ -131,6 +131,9 @@ OSD_Entry menuOsdActiveElemsEntries[] =
 #ifdef USE_OSD_STICK_OVERLAY
     {"STICK OVERLAY LEFT", OME_VISIBLE, NULL, &osdConfig_item_pos[OSD_STICK_OVERLAY_LEFT], DYNAMIC},
     {"STICK OVERLAY RIGHT",OME_VISIBLE, NULL, &osdConfig_item_pos[OSD_STICK_OVERLAY_RIGHT], DYNAMIC},
+#endif
+#ifdef USE_OSD
```

Comment:
This seems unnecessary - the entire thing is already wrapped in `#if defined(USE_OSD) && defined(USE_CMS)`

Reply:
Fixed.